### DISCUSSION TITLE
Need of destination account for transactions

### DISCUSSION THREAD

**Discussion by JSchimmelpfennig** [firefly-iii/firefly-iii#8805] [Firefly III]
Hello :) First, thank you for this amazing project.
I'm coming from MyMicroBalance and the workflow is there:

Creating an expense, selecting the category (food, transportation, ...) => money is gone from the selected source account.

But here in Firefly III I have to select a destination account and I'm confused why this is needed and if that is like a category for expenses.

I read this article:
https://docs.firefly-iii.org/references/firefly-iii/transaction-types/

"To create a withdrawal select a source account first. This should be an asset account or a liability. Then select a destination account (an expense account) or free-type a new one."

I also read this discussion: https://github.com/orgs/firefly-iii/discussions/8646

I still don't understand the purpose and the usage of the destination account when doing a withdrawal. 

Thank you in advance for clearing up my confusion :)

**Comment by JC5** [comment]
Thanks! 

The destination account is like the name of the shop or the place you bought something. It's up to you if you want to fill it in.

**Comment by JSchimmelpfennig** [comment]
So for categorizing should I use the destination account or the categories or tags? What's best practice for analyzing my expenses later?

**Reply by JC5** [reply]
Doesn't really matter, both can be used :)
